.github/workflows/test.yml

```
@@ -11,7 +11,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python-version: ["3.9", "3.10", "3.11", "3.12"]
+        python-version: ["3.10", "3.11", "3.12"]
```

Maybe we should now add py3.13
```suggestion
        python-version: ["3.10", "3.11", "3.12", "3.13]
```